adcc/backends/test_backends_crossref.py

```diff
@@ -50,6 +50,18 @@ def template_adc2_h2o(self, basis):
             results[b] = adcc.adc2(scfres, n_singlets=5, conv_tol=1e-10)
         compare_adc_results(results, 5e-9)
 
+    def template_adc2_uhf_h2o(self, basis):
+        results = {}
+        # UHF not supported for VeloxChem
+        if "veloxchem" in backends:
+            backends.remove("veloxchem")
+            if not len(backends):
+                pytest.skip("Not enough backends that support UHF available.")
```

Comment:
The indention level here is wrong, right?

Reply:
@maxscheurer You're call :smile:

Reply:
Actually ... this could be made `ch2nh2` as well, don't you think?

Reply:
The indent level at this point does not matter for the code to work properly, but it's more clear if lines 58/59 were out-dented once.

`ch2nh2` is okay for me here.

Reply:
Done.